src/include/units/bits/unit_text.h

```diff
@@ -29,13 +29,19 @@ namespace units::detail {
 template<typename Ratio>
 constexpr auto ratio_text()
 {
-  if constexpr(Ratio::num != 1 || Ratio::den != 1) {
+  if constexpr(Ratio::num != 1 || Ratio::den != 1 || Ratio::exp != 0) {
     auto txt = basic_fixed_string("[") + regular<Ratio::num>();
-    if constexpr(Ratio::den == 1) {
+    if constexpr(Ratio::den == 1 && Ratio::exp == 0) {
       return txt + basic_fixed_string("]");
     }
+    else if constexpr (Ratio::den == 1) {
+      return txt + basic_fixed_string(" x 10") + superscript<Ratio::exp>() +
+          basic_fixed_string("]");
+    }
```

Comment:
```suggestion
    if constexpr(Ratio::den == 1) {
      if constexpr(Ratio::exp == 0) {
        return txt + basic_fixed_string("]");
      }
      else {
        return txt + basic_fixed_string(" × 10") + superscript<Ratio::exp>() +
            basic_fixed_string("]");
      }
    }
```

Please note that I used Unicode `×` character above.

Reply:
OK, I have used a unicode literal though, to make it obvious and avoid non ascii literals in code